library/tomee

```diff
@@ -1,193 +1,164 @@
 GitRepo: https://github.com/tomitribe/docker-tomee.git
-GitCommit: 3e4e2b546fbba84a4559743a1d35ddab7de3fae0
+GitCommit: 47f8011e6567dfc4317d7f5b84e9aafb14f5963d
 Architectures: amd64, arm64v8
 
-Tags: 8.0.16-jre17-Temurin-ubuntu-webprofile, 8.0.16-Temurin-ubuntu-webprofile, 8.0.16-jre17-ubuntu-webprofile, 8.0.16-ubuntu-webprofile, 8.0.16-jre17-Temurin-webprofile, 8.0.16-Temurin-webprofile, 8.0.16-jre17-webprofile, 8.0.16-webprofile, 8.0.16-jre17-Temurin-ubuntu, 8.0.16-Temurin-ubuntu, 8.0.16-jre17-ubuntu, 8.0.16-ubuntu, 8.0.16-jre17-Temurin, 8.0.16-Temurin, 8.0.16-jre17, 8.0.16, 8.0-jre17-Temurin-ubuntu-webprofile, 8.0-Temurin-ubuntu-webprofile, 8.0-jre17-ubuntu-webprofile, 8.0-ubuntu-webprofile, 8.0-jre17-Temurin-webprofile, 8.0-Temurin-webprofile, 8.0-jre17-webprofile, 8.0-webprofile, 8.0-jre17-Temurin-ubuntu, 8.0-Temurin-ubuntu, 8.0-jre17-ubuntu, 8.0-ubuntu, 8.0-jre17-Temurin, 8.0-Temurin, 8.0-jre17, 8.0, 8-jre17-Temurin-ubuntu-webprofile, 8-Temurin-ubuntu-webprofile, 8-jre17-ubuntu-webprofile, 8-ubuntu-webprofile, 8-jre17-Temurin-webprofile, 8-Temurin-webprofile, 8-jre17-webprofile, 8-webprofile, 8-jre17-Temurin-ubuntu, 8-Temurin-ubuntu, 8-jre17-ubuntu, 8-ubuntu, 8-jre17-Temurin, 8-Temurin, 8-jre17, 8, jre17-Temurin-ubuntu-webprofile, Temurin-ubuntu-webprofile, jre17-ubuntu-webprofile, ubuntu-webprofile, jre17-Temurin-webprofile, Temurin-webprofile, jre17-webprofile, webprofile, jre17-Temurin-ubuntu, Temurin-ubuntu, jre17-ubuntu, ubuntu, jre17-Temurin, Temurin, jre17, latest
-Directory: TomEE-8.0/jre17/Temurin/ubuntu/webprofile
+Tags: 9.1.3-jre11-Temurin-ubuntu-microprofile, 9.1.3-Temurin-ubuntu-microprofile, 9.1.3-jre11-ubuntu-microprofile, 9.1.3-ubuntu-microprofile, 9.1.3-jre11-Temurin-microprofile, 9.1.3-Temurin-microprofile, 9.1.3-jre11-microprofile, 9.1.3-microprofile, 9.1.3-jre11-Temurin-ubuntu, 9.1.3-Temurin-ubuntu, 9.1.3-jre11-ubuntu, 9.1.3-ubuntu, 9.1.3-jre11-Temurin, 9.1.3-Temurin, 9.1.3-jre11, 9.1.3, 9.0-jre11-Temurin-ubuntu-microprofile, 9.0-Temurin-ubuntu-microprofile, 9.0-jre11-ubuntu-microprofile, 9.0-ubuntu-microprofile, 9.0-jre11-Temurin-microprofile, 9.0-Temurin-microprofile, 9.0-jre11-microprofile, 9.0-microprofile, 9.0-jre11-Temurin-ubuntu, 9.0-Temurin-ubuntu, 9.0-jre11-ubuntu, 9.0-ubuntu, 9.0-jre11-Temurin, 9.0-Temurin, 9.0-jre11, 9.0, 9-jre11-Temurin-ubuntu-microprofile, 9-Temurin-ubuntu-microprofile, 9-jre11-ubuntu-microprofile, 9-ubuntu-microprofile, 9-jre11-Temurin-microprofile, 9-Temurin-microprofile, 9-jre11-microprofile, 9-microprofile, 9-jre11-Temurin-ubuntu, 9-Temurin-ubuntu, 9-jre11-ubuntu, 9-ubuntu, 9-jre11-Temurin, 9-Temurin, 9-jre11, 9, jre11-Temurin-ubuntu-microprofile, Temurin-ubuntu-microprofile, jre11-ubuntu-microprofile, ubuntu-microprofile, jre11-Temurin-microprofile, Temurin-microprofile, jre11-microprofile, microprofile, jre11-Temurin-ubuntu, Temurin-ubuntu, jre11-ubuntu, ubuntu, jre11-Temurin, Temurin, jre11, latest
```

Comment:
This (and following lines) have a lot of `9.0` in them that I think should be `9.1`? (it's hard to see because there's just _so many_ tags, but I think that's what we're seeing in the sorted tag list in our diff :+1:)

Reply:
@tianon excellent catch.   One day I'll get tags right the 1st time. lol.

Incoming commit.

Reply:
Might be worth scripting, if you're doing these updates by hand :sweat_smile:

(I know I'd sure be nervous if I were maintaining this matrix by hand! :joy:)

Reply:
I have a script that gets me 90% of they way.  I just  neglected to change 9.0 to 9.1.  Sometimes it's the simple things.